src/args.rs

```diff
@@ -150,3 +156,56 @@ pub fn parse_args() -> crate::MiniserveConfig {
         file_upload: args.file_upload,
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use rstest::rstest_parametrize;
+
+    fn create_required_auth(username: &str, password: &str, encrypt: &str) -> auth::RequiredAuth {
```

Comment:
You should describe the semantics of this function. Something like `Helper function that ...`.